packages/user-interface/src/components/PageHeader.tsx

```diff
@@ -10,13 +11,15 @@ interface Props {
 }
 
 const PageHeader = ({ loading, title, subTitle, children }: Props) => {
+  const { getNotifications } = useNotification();
   if (loading) return <Skeleton height={40} width={250} />;
   if (!title && !children) return null;
 
   return (
     <header className={styles["page-header"]}>
```

Comment:
Hier misschien nog de rich text toevoegen om de spacing te kunnen bepalen?

Reply:
Dit slaan we nu nog even over.